vnet/spec/vnet/endpoints/1.0/networks_spec.rb

```diff
@@ -21,6 +21,31 @@ def app
   include_examples 'GET /:uuid'
   include_examples 'DELETE /:uuid'
 
+  describe 'DELETE /:uuid' do
```

Comment:
How would that work since segments don't have a relation to ip_leases? Btw, I had made another commit that moves this test to spec/models instead of spec/endpoints and now I realize I'm a complete idiot and never pushed it. :( I hope I still have it locally.